src/app/common/money/calculate-money.ts

```diff
@@ -31,6 +31,18 @@ export function convertAmountToFractionalUnit(num: Money | BigNumber, decimals?:
   return num.shiftedBy(decimals);
 }
 
+export function convertToMoneyTypeWithDefaultOfZero(
+  symbol: string,
+  num?: NumType,
+  decimals?: number
+) {
```

Comment:
You can use `Money` here ?

```suggestion
export function convertToMoneyTypeWithDefaultOfZero({
  symbol,
  amount = new BigNumber(0),
  decimals
}: Money) {
```

Reply:
I can look at this, just relocated. 👍

Reply:
Not totally following this suggestion considering the method is to convert to Money type, but you want to pass in Money type?